react-native-git-upgrade/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-native-git-upgrade",
+  "version": "0.1.0",
+  "license": "BSD-3-Clause",
+  "description": "The React Native upgrader tool",
+  "main": "cli.js",
+  "bin": {
+    "react-native-git-upgrade": "index.js"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/react-native.git"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "babel-core": "^6.18.0",
+    "babel-preset-es2015-node": "^6.1.1",
+    "babel-preset-stage-3": "^6.17.0",
+    "babel-register": "^6.18.0",
+    "npmlog": "^4.0.0",
+    "promise": "^7.1.1",
+    "rimraf": "^2.5.4",
+    "semver": "^5.0.3",
+    "shelljs": "^0.7.5",
+    "yeoman-environment": "1.5.3"
```

Comment:
I wonder if it would be possible to simply call into the CLI using `require('node_modules/react-native/upgrade')` or similar and calling it. That way you might not need to declare this dependency here since react-native already depends on it?